lib/ansible/utils/module_docs_fragments/utm.py

```diff
@@ -35,10 +40,8 @@ class ModuleDocFragment(object):
           - The desired state of the object.
           - C(present) will create or update an object
           - C(absent) will delete an object if it was present
-          - C(info) will return the object details
```

Comment:
If you are going to remove this then you might want to remove [this](https://github.com/ansible/ansible/blob/59dcf3f27743ecbecfadc037515d0fa33ae3045a/lib/ansible/module_utils/utm_utils.py#L67) as well.